lib/protobuf/nats/config.rb

```diff
@@ -40,11 +41,12 @@ def load_from_yml(reload = false)
             config_path = ENV["PROTOBUF_NATS_CONFIG_PATH"] || ::File.join("config", "protobuf_nats.yml")
             absolute_config_path = ::File.expand_path(config_path)
             if ::File.exist?(absolute_config_path)
```

Comment:
Not in the scope of MR,  I thought `unless` is the preferred condition check but we are using `if file.exist?`